src/main/java/io/anserini/encoder/dense/CosDPRDistilEncoder.java

```diff
@@ -0,0 +1,71 @@
+package io.anserini.encoder.dense;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import ai.djl.modality.nlp.DefaultVocabulary;
+import ai.djl.modality.nlp.bert.BertFullTokenizer;
+import ai.onnxruntime.OnnxTensor;
+import ai.onnxruntime.OrtEnvironment;
+import ai.onnxruntime.OrtException;
+import ai.onnxruntime.OrtSession;
+
+/**
+ * CosDPRDistil
+ */
+public class CosDPRDistilEncoder extends DenseEncoder {
```

Comment:
I think this class should be named `CosDprDistilEncoder`? Yea, I know... looks ugly... but that's the Java camel case convention...